web/src/app/pages/create-survey/task-details/task-details.component.html

```diff
@@ -15,5 +15,29 @@
 -->
 
 <div id="task-details">
-    <h1>task-details works!</h1>
-</div>
\ No newline at end of file
+  <h1>Define how specified data should be collected</h1>
+  <h2>
+    Data collectors will be prompted to complete the tasks you define here in
+    order.
+  </h2>
+
+  <div id="tasks-added">
+    <task-input
+      *ngFor="let task of tasks; let i = index; trackBy: trackByFn"
```

Comment:
Nit: Please name the `trackByFn` something more descriptive here any throughout, eg: `getIndex()`

Reply:
Done!